src/main/webapp/scripts/forum-script.js

```diff
@@ -1,15 +1,24 @@
 import {forumTemplate} from './forum-template.js';
 
-$( document ).ready( getFilters );
-$( document ).ready( getForum );
+$( document ).ready( function() {
+  getForum();
+  getFilters();
+  $('#search-button').click(search);
+  $('#search-button').keyup(function(event) {
+    if (event.keyCode === 13) {
```

Comment:
Can you use a constant for 13? A literal 13 looks pretty ominous :)

Reply:
Yeah totally missed that.  I created a const in the method: enterKey.